src/components/backend-ai-import-view.ts

```diff
@@ -279,6 +280,50 @@ export default class BackendAIImport extends BackendAIPage {
     }
   }
 
+  getGitLabRepoFromURL() {
+    let url = this.shadowRoot.querySelector('#gitlab-repo-url').value;
+    let tree = 'master';
+    let get_branch_name = this.shadowRoot.querySelector('#gitlab-default-branch-name').value;
```

Comment:
Let's use the camel-case variable name if there is no other reason. The syntax should automatically be checked with some linting tools, but it is disabled for now.